"failed to stop infra container" in rootless tests

"Report every CI issue" -- well, here we go...
```
INFO: Running rootless tests
bash -f integration/rootless/rootless_test.sh
a569f2de7a831c271aec1a9769d2ec309ebbf70a12c2aa1946441e5d85c516aa
time="2021-09-28T14:55:16Z" level=fatal msg="stopping the pod sandbox \"a569f2de7a831c271aec1a9769d2ec309ebbf70a12c2aa1946441e5d85c516aa\": rpc error: code = Unknown desc = failed to stop infra container for pod sandbox a569f2de7a831c271aec1a9769d2ec309ebbf70a12c2aa1946441e5d85c516aa: failed to get container 'stopped' status k8s_POD_nginx-sandbox_default_hdishd83djaidwnduwk28bcs3_1: failed to get container stopped status: not found"
[rootless_test.sh:33] ERROR: sudo crictl stopp a569f2de7a831c271aec1a9769d2ec309ebbf70a12c2aa1946441e5d85c516aa
make: *** [Makefile:177: rootless] Error 1
[run.sh:72] ERROR: sudo -E PATH=/tmp/jenkins/workspace/kata-containers-2.0-debian-PR/go/bin:/usr/local/go/bin:/usr/sbin:/sbin:/usr/local/bin:/usr/bin:/bin:/usr/games bash -c make rootless
[jenkins_job_build.sh:193] ERROR: ci/run.sh
++ handle_error 84
```
in http://jenkins.katacontainers.io/job/kata-containers-2.0-debian-PR/2194

--- a/integration/rootless/rootless_test.sh
+++ b/integration/rootless/rootless_test.sh
@@ -26,12 +26,13 @@ setup() {
 
 cleanup() {
 	sudo sed -i -e 's/^.*\(rootless\)/# \1/g' /usr/share/defaults/kata-containers/configuration.toml
+	sudo crictl stopp $(sudo crictl pods -q) || true
+	sudo crictl rmp $(sudo crictl pods -q) || true
 }
 
 run() {
 	sudo crictl runp -r kata "${dir_path}/rootless-pod.json"
-	sudo crictl stopp $(sudo crictl pods -q)
-	sudo crictl rmp $(sudo crictl pods -q)
+	waitForProcess 15 3 "sudo crictl inspectp $(sudo crictl pods -q) | jq '.status.state' | grep 'SANDBOX_READY'"
 }
 
 main() {

